mysql-test/suite/wsrep/my.cnf

```diff
@@ -2,10 +2,17 @@
 !include include/default_mysqld.cnf
 
 [mysqld.1]
-wsrep_provider=@ENV.WSREP_PROVIDER
+binlog-format=row
+innodb_autoinc_lock_mode=2
+innodb_flush_log_at_trx_commit=2
+
+# wsrep variables
+wsrep_causal_reads=ON
 wsrep_cluster_address='gcomm://'
 wsrep_node_address=127.0.0.1
-binlog-format=row
+wsrep_node_name=node1
+wsrep_provider=@ENV.WSREP_PROVIDER
+wsrep_sync_wait=15
```

Comment:
why did you add the wsrep_sync_wait change to suite cnf? wouldn't it affect entire suite?

Reply:
This is the first test to go into the `wsrep` suite in 8.0. So, we are safe :)

Reply:
The reason for adding it to the cnf is to get the consistent reads in the test cases so that it becomes easier to write tests thinking about synchronization of previous writes.
```
--connection node_1
INSERT INTO t1 VALUES(1);

--connection node_2
SELECT * FROM t1;
```

Reply:
ok, 15 comes from this. So we do cluster-sync check on these SQLs.
```
enum enum_wsrep_sync_wait {
    WSREP_SYNC_WAIT_NONE = 0x0,
    // select, begin
    WSREP_SYNC_WAIT_BEFORE_READ = 0x1,
    WSREP_SYNC_WAIT_BEFORE_UPDATE_DELETE = 0x2,
    WSREP_SYNC_WAIT_BEFORE_INSERT_REPLACE = 0x4,
    WSREP_SYNC_WAIT_BEFORE_SHOW = 0x8,
    WSREP_SYNC_WAIT_MAX = 0xF
  };
```

Reply:
@venkatesh-prasad-v if this is new suite, I would rename it to wsrep_one_node or wsrep_1node

Reply:
@venkatesh-prasad-v do you know why we are different from upstream for this variable? upstream supports an additional value of 3.
https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-sync-wait

Reply:
@satya-bodapati 
> if this is a new suite, I would rename it to wsrep_one_node or wsrep_1node
wsrep suite is actually meant for one node only. So it is implicit and there is no need to add any extension.


>do you know why we are different from upstream for this variable? upstream supports an additional value of 3.

We also support value 3 (since it is bitmask internally) . It has also been documented well in https://www.percona.com/doc/percona-xtradb-cluster/8.0/wsrep-system-index.html#wsrep_sync_wait

```
mysql> set wsrep_sync_wait=3;
Query OK, 0 rows affected (0.00 sec)

mysql> select @@wsrep_sync_wait;
+-------------------+
| @@wsrep_sync_wait |
+-------------------+
|                 3 |
+-------------------+
1 row in set (0.00 sec)
```

Reply:
>With wsrep_sync_wait set to 15, we can always ensure that SELECT on node_2 is executed only after INSERT is replicated and applied on node_2

Wasn't the intention of this suite to use only 1 node?